tests/e2e-automatic-rbac/receiver-k8sevents/01-install.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: simplest
+  namespace: chainsaw-k8s-events
+spec:
+  config: |
+    receivers:
+      k8s_events:
+    processors:
+    exporters:
+      debug:
+    service:
+      pipelines:
+        traces:
```

Comment:
This shouldn't work and should fail with `Error: failed to build pipelines: failed to create "k8s_events" receiver for data type "traces": telemetry type is not supported`.

I wonder if this is the reason why no CR/CRB is created if I deploy exactly this manifest, but with `logs` instead of `traces`?

Reply:
Nevermind, my operator SA was missing permissions... but this typo is legit.

Reply:
Worth fixing, but for the purpose of this test it doesn't matter. We don't even care if the collector starts, just that the generated ClusterRole is correct.